core/tool_roundtrip.c

```diff
@@ -567,6 +567,66 @@ _roundtrip_put_from_file (const char * const * properties)
 	oio_error_pfree (&err);
 }
 
+/* Upload a content larger than the chunk size
+ * then check the max chunk size sent */
+static void
+put_reduced_chunksize (const char * const * properties)
+{
+	const gsize chunk_size = 5*1024*1024;
+	struct oio_error_s *err = NULL;
+
+	struct oio_url_s *url_random = oio_url_dup(url);
+	gchar path[32];
+	oio_str_randomize(path, oio_ext_rand_int_range(7,32), random_chars);
+	oio_url_set (url_random, OIOURL_PATH, path);
```

Comment:
In this file, indentation is a not very consistent, but I recommend not putting a space between a function name and an opening parenthesis.